tools/README.md

```diff
@@ -28,6 +28,38 @@ Running K8S API test
 pod "diag" deleted
 ```
 
+## K8S Template generator
+
+Docker based:
+
+```bash
+docker run --rm kubernetes-tools \
+  template \
+  --namespace <NAMESPACE> \
+  --name collection \
+  --set sumologic.accessId='<ACCESS_KEY>' \
+  --set sumologic.accessKey='<ACCESS_ID>' \
+  --set sumologic.collectorName='<COLLECTOR_NAME>' \
+  --set sumologic.clusterName='<CLUSTER_NAME>' \
+  | tee setup-sumologic.yaml
+```
+
+Kubernetes based:
+
+```
+kubectl run tools --generator=run-pod/v1 -it --rm --restart=Never --image sumologic/kubernetes-tools -- \
+  template \
+  --namespace <NAMESPACE> \
+  --name collection \
+  --set sumologic.accessId='<ACCESS_KEY>' \
+  --set sumologic.accessKey='<ACCESS_ID>' \
+  --set sumologic.collectorName='<COLLECTOR_NAME>' \
+  --set sumologic.clusterName='<CLUSTER_NAME>' \
+  | tee setup-sumologic.yaml
+```
+
+As the result you will get `setup-sumologic.yaml` which is ready to apply for kubernetes.
```

Comment:
the name of `setup-sumologic.yaml` is already used to refer to the Sumo collector & sources setup using terraform (see under `/deploy/kubernetes/setup-sumologic.yaml.tmpl`)
I think we should change the name of the generated yaml file here, maybe to `sumologic.yaml` or something more generic to encapsulate the fact that this is for the entire chart